libs/jsonschema/from_type_test.go

```diff
@@ -11,17 +11,17 @@ import (
 
 func TestFromTypeBasic(t *testing.T) {
 	type myStruct struct {
-		S             string `json:"s"`
-		I             *int   `json:"i,omitempty"`
-		V             any    `json:"v,omitempty"`
-		TriplePointer ***int `json:"triple_pointer,omitempty"`
+		S                string `json:"s"`
+		I                *int   `json:"i,omitempty"`
+		V                any    `json:"v,omitempty"`
+		TriplePointer    ***int `json:"triple_pointer,omitempty"`
+		DeprecatedTagged string `json:"deprecated_tagged,omitempty" bundle:"deprecated"`
```

Comment:
I'd consider removing the `bundle:"deprecated"` tag altogether since it's no longer functional. WDYT?

Reply:
@shreyas-goenka Yeah I had the same thought that maybe that tag is no longer needed but I didn't have a full enough understanding to know if that was actually the case.

I'll try removing it 👍